### PR TITLE
[airbyte-ci] format: include .md files

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

Now that #37839 is merged, we can start linting Markdown in `format fix all`!

I want to make sure we have a doc that advises on what editor settings to use. For example for Markdown, most editors would pick up format-on-save with `.prettierrc` that we already have, and it has valid ignores setup already.

### CODE REVIEW

**Review by bnchrch — CHANGES_REQUESTED** [review]
Im all for this! but can we run the linter and commit the changes in the same PR :)

**Review by aaronsteers — APPROVED** [review]

**Comment by natikgadzhi** [comment]
Wow, this is interesting! I can _swear_ I just merged the original PR where I setup prettier + prettierignore and it was green. BRB, verifying that `airbyte-ci format fix js` and `prettier -w **/*.md` do the same / similar thing, and pushing it up.

**Comment by natikgadzhi** [comment]
https://www.loom.com/share/4eb57f8a66ef49dabf5f26f65f6080a3 🙃 

This is insane.

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/configuration.py [hunk 1]
```diff
@@ -35,9 +35,11 @@ class FormatConfiguration:
     # Run prettier on all json and yaml files.
     FormatConfiguration(
         Formatter.JS,
-        ["**/*.json", "**/*.yaml", "**/*.yml"],
+        ["**/*.json", "**/*.yaml", "**/*.yml", "**/*.md"],
         format_js_container,
-        [f"prettier --write . --list-different --cache --cache-location={CACHE_MOUNT_PATH}/.prettier_cache"],
+        [
```
Comment (natikgadzhi) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/configuration.py:40]: I still get a mismatch locally, when I run airbyte-ci format fix js, it does NOT see those files that are in this PR. I had to run prettier locally instead. I don't know what's borked in my local dagger, but I'm very curious. Not a high-leverage work item anymore. /cc @alafanechere

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py [hunk 2]
```diff
@@ -98,14 +98,16 @@ def format_java_container(dagger_client: dagger.Client, java_code: dagger.Direct
     )
 
 
-def format_js_container(dagger_client: dagger.Client, js_code: dagger.Directory, prettier_version: str = "3.0.3") -> dagger.Container:
+def format_js_container(dagger_client: dagger.Client, js_code: dagger.Directory, prettier_version: str = "3.2.5") -> dagger.Container:
     """Create a Node container with prettier installed with mounted code to format and a cache volume."""
+    warmup_dir = dagger_client.host().directory(".", include=WARM_UP_INCLUSIONS[Formatter.JS], exclude=DEFAULT_FORMAT_IGNORE_LIST)
```
Comment (natikgadzhi) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py:103]: I am okay removing these changes — this seems to have no effect, but I don't see how this hurts either. Fine either way.

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/configuration.py (hunk 1 above)
Comment (alafanechere) [reply airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/configuration.py:40]: It'd be interesting to set a breakpoint [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/format_command.py#L87) to double check if markdown files are mounted to the `dir_to_format`.
Did you run the command from the root of the repo?
Did you check that [`DEFAULT_FORMAT_IGNORE_LIST`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py#L12) does not override your filters?

**Review by alafanechere — APPROVED** [review]

**Review by alafanechere — COMMENTED** [review]
Can you plase bump `airbyte-ci` version in `airbyte-ci/connectors/pipelines/pyproject.toml` and add a changelog entry to the README?

**Pull request closed without merging**